src/components/Suspense/SuspenseWithPreviousRenderAsFallback.tsx

```diff
@@ -10,5 +10,7 @@ import React, { Suspense } from 'react'
 
 export const SuspenseWithPreviousRenderAsFallback = (props: { children: React.ReactNode }) => {
   const previousContents = usePrevious(props.children)
-  return <Suspense fallback={previousContents ?? null}>{props.children}</Suspense>
+  const previousDeferred = useDeferredValue(previousContents)
```

Comment:
evidently this works? https://stackoverflow.com/a/76173194

Reply:
do we need `usePrevious` and `useDeferredValue` or would this work with just `useDeferredValue`?

Reply:
Confirmed that just using `useDeferredValue` gives the desired search behaviour without crashing on iPad

Reply:
```suggestion
  const previousDeferred = useDeferredValue(props.children)
```

Reply:
we may need previous, this area also fixes another issue we had in search, let me make sure tests pass and it works still without

Reply:
confirm it works this way!